src/app/tests/TestReadInteraction.cpp

```diff
@@ -76,41 +76,40 @@ static chip::System::Clock::ClockBase * gRealClock;
 class TestContext : public chip::Test::AppContext
 {
 public:
-    static int Initialize(void * context)
+    static int SetUp(void * context)
```

Comment:
Please document so it's clear what the difference between SetUp/TearDown and Initialize/Terminate is....

And maybe fine better names (e.g. StartTestSuite vs StartTest or something)?

Reply:
Changed to names compatible with Google Test instead of names from nlTest